ppl/archive/import.go

```diff
@@ -315,7 +315,8 @@ func (cw *chunkWriter) Write(rec *zng.Record) error {
 	if err := cw.dataFileWriter.Write(rec); err != nil {
 		return err
 	}
-	if cw.needIndexWrite {
+	ts := rec.Ts()
+	if !cw.wroteFirst || (cw.needIndexWrite && ts != cw.lastTs) {
```

Comment:
You could replace the `rec.Ts()` call below this line with `ts`.

Also, `needIndexWrite` is initialized to true inside `newChunkWriter`; with this check on `wroteFirst`, I don't think it needs to be set there anymore.

Reply:
Yeah, zap that.

Reply:
I don't quite follow the `wroteFirst` thing. You mean I don't need this here? The problem with that would be if the first Ts was 0, the first offset entry would not have 0 as the ts value.

Reply:
@alfred-landrum observation is just that there's no need for `needIndexWrite: true,` in newChunkWriter since you now have `!cw.writeFirst` here, and it's always true on the first trip through this method.

Reply:
ooooh, good observation!